uint8 force

uint8 mode
uint8 MODE_BASIC   = 0
uint8 MODE_PINCH   = 1
uint8 MODE_SCISSOR = 2
uint8 MODE_WIDE    = 3

uint8 position

uint8 speed